packages/entities/entities-plugins/src/components/free-form/Datakit/flow-editor/store/init.ts

```diff
@@ -264,22 +264,49 @@ export function collectConnectionsFromConfigNode(
 }
 
 /**
- * Resolve input and output fields for a node instance.
- * If the node type has configurable IO, use the UI node's fields if provided.
- * Otherwise, return undefined to fallback to the default fields from metadata.
+ * Resolves the input and output fields for a node instance.
+ *
+ * - If the node type supports configurable IO, prefer the UI node’s fields when available.
+ * - Otherwise, return `undefined` so the default fields from metadata are used.
+ * - If `unNode` is not provided (e.g. when importing examples or switching to the flow editor
```

Comment:
nit: Probably a typo

```suggestion
 * - If `uiNode` is not provided (e.g. when importing examples or switching to the flow editor
```